docs/source/en/migration.md

```diff
@@ -1 +1,300 @@
-this document is still waiting for translation, see [Chinese Version](/zh-cn/migration.html)
\ No newline at end of file
+title: Egg@2 Upgrade guideline
+---
+
+## Background
+
+With the official release of Node.js 8 LTS, egg now comes with built-in ES2017 Async Function support. 
+
+Though the TJ [co] has has brought `async/await` programming experience before this, but it also has some inevitable problems:
+
+- performance lost
+- [obscure error logs](https://github.com/eggjs/egg/wiki/co-vs-async)
+
+In the official Egg 2.x:
+- Full compatibility to Egg 1.x and `generator function`.
+- Koa 2.x based `async function` solutions. 
+- Only support Node.js 8 and above.
+- Better error stack messages without [co], approximately 30% performance improvement (do not include the performance improvement brought by Node), see [benchmark](https://eggjs.github.io/benchmark/plot/) for more details.
+
+One of the Egg's concept is `progressive`, hence we provide progressive programming experiences to developers.
+
+- [Quick upgrade](#quick_upgrade)
+- [Plugin update](#plugin)
+- [Further upgradation](#forther_upgradation)
+- [Upgrade documents for Plugin developers](#upgrade_documents_for_plugin_developers)
+
+## Quick upgrade
+- Use the latest Node LTS version (`>=8.9.0`).
+- Change `egg` version to `^2.0.0` in `package.json`.
+- Check if included plugins are the latest version (optional).
+- Reinstall the dependencies, and run unit tests again.
+
+**Done! Upgrade finished without any code changes.**
```

Comment:
`barely with any code changes` is more accurate.

Reply:
Updated, it's been changed to
`Done! Barely with any code changes.`